clients/cobol-lsp-vscode-extension/package.json

```diff
@@ -54,6 +54,10 @@
             {
                 "command": "cobol-lsp.cpy-manager.goto-settings",
                 "title": "Open Copybook location settings"
+            },
+            {
+                "command": "cobol-lsp.action.commentLine",
+                "title": "Comment line"
```

Comment:
Could we rename the title to `Toggle COBOL Line Comment`? That would make it more in line with the default `Toggle Line Comment`. We are also going to implement commenting for HLASM and I would call it `Toggle HLASM Line Comment` there to have some consistency if a user searches the command pallet.

Reply:
Sure. Done.